scripts/linux/trcE2ETest.sh

```diff
@@ -870,9 +872,10 @@ function configure_connectivity_settings() {
     NETWORK_CONTROLLER_RUNPROFILE=${NETWORK_CONTROLLER_RUNPROFILE:-Offline}
     TEST_DURATION="${TEST_DURATION:-01:00:00}"
 
+    VERIFICATION_DELAY="${VERIFICATION_DELAY:-00:15:00}"
+
     # Needs to be high due to 1ES conncectivity issues.
-    # If this is ever resolved, can be bumped back to 15 mins.
-    VERIFICATION_DELAY="${VERIFICATION_DELAY:-00:45:00}"
+    CHECK_TRC_DELAY="${TEST_START_DELAY:-00:30:00}"
```

Comment:
Please add a comment that explains 'why' these two variables {`VERIFICATION_DELAY`, `CHECK_TRC_DELAY`} are needed and how are they different. 

Nit: `CHECK_TRC_DELAY` sounds misleading. Maybe we can name it something like... `1ES_AGENT_NETWORKING_DELAY`?

Reply:
1. Will add better comment, good idea.
2. 1ES_AGENT_NETWORKING_DELAY indicates why it was there, but I'd rather have a name that indicates what the delay is doing. The other names follow this pattern, so I'd rather go with CHECK_TRC_DELAY.
3. Yes my intention was to bump test start delay to 10 mins.

Reply:
Regarding 2, I think the comment you suggested will be the best of both worlds